packages/convex-helpers/server/index.ts

```diff
@@ -203,3 +203,39 @@ export function crud<
     >,
   };
 }
+
+export type JSONValue =
+  | null
+  | boolean
+  | undefined
+  | number
+  | string
+  | JSONValue[]
+  | { [key: string]: JSONValue };
+
+/**
+ * Returns a Response object with the data as JSON and
+ * correct Content-Type header, for use with httpAction.
+ *
+ * Used like:
+ * ```ts
+ * http.route({
+ *   path: "/templates",
+ *   method: "GET",
+ *   handler: httpAction(async (ctx) => {
+ *     // ...
+ *     return json({ message: "Hello, world!" });
+ *   }),
+ * });
+ * ```
+ *
+ * @param data The data you want to return as JSON.
+ * @returns A Response object with the data as JSON.
+ */
+export function json(data: JSONValue) {
+  return new Response(JSON.stringify(data), {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Response/json_static exists (and is implemented). But I think the types only exist in newer versions of TypeScript.

Would prefer pointing to that instead of creating our own version